### PR TITLE
A4A > Agency Tier: Implement current agency tier levels

### PR DESCRIPTION
**Pull request by yashwin**
Closes https://github.com/Automattic/automattic-for-agencies-dev/issues/1294

## Proposed Changes

This PR implements the current agency tier section.

* To showcase the current agency tier.

## Testing Instructions

* Open the A4A live link
* Update your agency tier to **Emerging Partner** using the MC tool > Go to Agency Tier in the UI > Verify the screen looks as shown below:

Screenshot 2024-10-16 at 3 18 07 PM

* Change the agency tier to **Agency Partner** > Refresh the UI and verify the screen looks as shown below:

Screenshot 2024-10-16 at 3 18 19 PM

* Change the agency tier to **Pro Agency Partner** > Refresh the UI and verify the screen looks as shown below:

Screenshot 2024-10-16 at 3 18 35 PM

* Verify it looks good on all the screen sizes:

Screenshot 2024-10-16 at 3 26 55 PM

Screenshot 2024-10-16 at 3 27 12 PM

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

**Review by madebynoam — APPROVED** [review]
Tested now. Looks great, @yashwin!

File: client/a8c-for-agencies/sections/agency-tier/primary/agency-tier-overview/style.scss [hunk 1]
```diff
@@ -179,6 +181,12 @@ $dark-green: #104e30;
 	}
 }
 
+.agency-tier-overview__top-content-right {
+	@include break-wide {
+		margin-inline-start: 12px;
```
Comment (jkguidaven) [inline client/a8c-for-agencies/sections/agency-tier/primary/agency-tier-overview/style.scss:186]: I’m considering that using a gap within `.agency-tier-overview__top-content @include break-wide` might be a better option and would eliminate the need to create a separate CSS rule for this div box.

File: client/a8c-for-agencies/style.scss [hunk 2]
```diff
@@ -642,3 +642,7 @@ html {
 		}
 	}
 }
+
+.opacity-50 {
```
Comment (jkguidaven) [inline client/a8c-for-agencies/style.scss:646]: I suggest prefixing `.is-*` here if this is a modifier to follow Calypso CSS guideline.

**Review by jkguidaven — APPROVED** [review]
Tested it and it works great. Left few non-blocking comments. LGTM

**Comment by yashwin** [comment]
Thanks for the review, @jkguidaven!

As discussed, I'll address your comments on another PR.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/16921030

Some locales (Hebrew, Japanese) have been temporarily machine-translated due to translator availability. All other translations are usually ready within a few days. Untranslated and machine-translated strings will be sent for translation next Monday and are expected to be completed by the following Friday.

Thank you @yashwin for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/16921030).

File: client/a8c-for-agencies/sections/agency-tier/primary/agency-tier-overview/style.scss (hunk 1 above)
Comment (yashwin) [reply client/a8c-for-agencies/sections/agency-tier/primary/agency-tier-overview/style.scss:186]: We already have a gap, which is being used for inline space. So, that wouldn't work here.

File: client/a8c-for-agencies/style.scss (hunk 2 above)
Comment (yashwin) [reply client/a8c-for-agencies/style.scss:646]: Fixed it here: https://github.com/Automattic/wp-calypso/pull/95548

**Pull request merged**